### ISSUE TITLE
Project Card Converted to Issue

### ISSUE THREAD

**Issue by LiamMahoney** [LiamMahoney/scrummy#21]
When a project card is converted into an issue, the proper `project: <project>` and `stage: <stage>` labels should be added to the new issue.

This is when the plus button is clicked on a project column (to create a project card) and then the ellipsis are clicked and `convert to issue` is selected.

### Hook
**hook:** project_card
**action:** converted

#### Adding the `project: <project>` label

- [x] reuse `issueAddedToProject` method

#### Adding the `stage: <stage>` label

- [x] figure out the column the project card is in 

- [x] find the matching `stage: <stage>` label

- [x] add the matching label to the issue

**Comment by LiamMahoney** [comment]
if the payload data is the same this could reuse the method issueAddedToProject method

**Comment by LiamMahoney** [comment]
Going to have to create a new state that calls issueAddedToProject method since issueAddedToProject isn't the only functionality needed here.

**Comment by LiamMahoney** [comment]
Noticed a refactor would be possible of the methods getProjectLabel and matchProjectLabel into a method that would take the 'type' of label to search for. So now I can use the same methods getLabel and matchLabel to look up the matching `stage: <stage>` or `project: <project>` label.

The refactor should be tested to make sure this didn't break anything with issue added to project and issue removed from project.

It has been tested and the old functionality is still working.

**Comment by LiamMahoney** [comment]
Noticed that the findLabel method should not be getting issue information. The issue information it retrieves is not used at all within the method. 

Issue information can and should be retrieved by its caller instead.

This refactor has been done. Tested and all of the old functionality is still working.

**Comment by LiamMahoney** [comment]
Reverted the refactor I had made on findProjectLabel due to this method needing to get the project / column information. However, I noticed that If I create a generic `getGithubURL` method I could make the methods `getProjectLabel` and `getStageLabel` into a more generic method. Going to create a separate issue for this at some point. For now, it's a FIXME comment in the `states.Issue.findStageLabel `method.
